gratipay/security/__init__.py

```diff
@@ -51,6 +51,7 @@ def add_headers_to_response(response):
                                                        'style-src assets.gratipay.com cloud.typography.com;'
                                                        'img-src *;'
                                                        'font-src assets.gratipay.com cloud.typography.com;'
+                                                       "script-src 'unsafe-inline'"
```

Comment:
We're missing a semicolon at the end here, ya? Addressed in 75ce62fbd4ac9526e0654c14b7b06d186e465511.